sources/packages/web/src/components/NavBar.vue

```diff
@@ -37,7 +37,7 @@ export default class NavBar extends Vue {
   }
 
   .p-menubar-start {
```

Comment:
Suggestion/Question: Arent there standard primevue css classes that can be used here?